pkg/urbit/daemon/main.c

```diff
@@ -298,7 +298,7 @@ _main_getopt(c3_i argc, c3_c** argv)
            && u3_Host.ops_u.url_c == 0
            && u3_Host.ops_u.git == c3n ) {
     u3_Host.ops_u.url_c =
-      "https://bootstrap.urbit.org/urbit-" URBIT_VERSION ".pill";
+      "https://bootstrap.urbit.org/urbit-v" URBIT_VERSION ".pill";
```

Comment:
This naming-scheme change remains planned?

Reply:
Fine by me, we'll just need to take this into account if we push another hotfix this era.

Reply:
@jtobin, reminder that the pill needs to be renamed to account for this scheme.